assets/sass/_theme/sections/locations.sass

```
@@ -1,51 +1,75 @@
 .locations
-    &--list
-        .location + .location
+    .location
+        display: flex
+        flex-direction: column-reverse
+        gap: calc(var(--grid-gutter) / 2)
+        justify-content: flex-end
+        position: relative
+        + .location
             margin-top: $spacing-4
+        a
+            @include stretched-link(after)
+            text-decoration: none
+        .more
+            @include hover-translate-icon(after)
+            @include meta
+            @include icon(arrow-right-line, after)
+                margin-left: $spacing-2
+            margin-top: $spacing-1
+        .media
+            width: columns(6)
+            picture
+                aspect-ratio: 3/2
+                display: block
+            img
+                display: block
+                height: 100%
+                object-fit: cover
```

Ici tu peux réduire, maintenant que t'as plus la problématique du cadre gris, comme ceci : 

```suggestion
        .media
            width: 100%
            img
                aspect-ratio: 3/2
                object-fit: cover
```

Grâce au width: 100%, ton image va s'adapter à ton contenant, ce qui est plus safe ! 

Ça te permet de supprimer ton fix lignes 68 à 70 : 
```
                .media
                    width: 100%
```